config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
+  devise_for :users
   resources :recipes
-  resources :reviews
+  resources :reviews,
+  path: '',
```

Comment:
the path: ... and all that's below is related to users, so please switch line 2 and 4